providers/mlx5/libmlx5.map

```diff
@@ -6,3 +6,8 @@ MLX5_1.0 {
 		mlx5dv_init_obj;
 	local: *;
 };
+
+MLX5_1.1 {
+	global:
+		mlx5dv_create_cq_ex;
+} MLX5_1.0;
```

Comment:
It would be better to follow the convection of the rest of the library and call this MLX5_1.14

1.0 should have be 1.13

And make sure the library symlink has  1.14 in the name